bokehjs/src/lib/models/glyphs/wedge.ts

```diff
@@ -58,28 +59,19 @@ export class WedgeView extends XYGlyphView {
   }
 
   protected override _hit_point(geometry: PointGeometry): Selection {
-    let dist, sx0, sx1, sy0, sy1, x0, x1, y0, y1
+    let dist, sx0, sx1, sy0, sy1
     const {sx, sy} = geometry
     const x = this.renderer.xscale.invert(sx)
     const y = this.renderer.yscale.invert(sy)
 
     // check diameter first
-    const max_diameter = 2 * this.max_radius
-    if (this.model.properties.radius.units === "data") {
-      x0 = x - max_diameter
-      x1 = x + max_diameter
-
-      y0 = y - max_diameter
-      y1 = y + max_diameter
-    } else {
-      sx0 = sx - max_diameter
-      sx1 = sx + max_diameter
-      ;[x0, x1] = this.renderer.xscale.r_invert(sx0, sx1)
-
-      sy0 = sy - max_diameter
-      sy1 = sy + max_diameter
-      ;[y0, y1] = this.renderer.yscale.r_invert(sy0, sy1)
-    }
+    const s_max_radius = max(this.sradius)
```

Comment:
Expensive computations, like this one, should be moved to `_map_data()` (the name should be `max_sradius`).

Reply:
Makes sense. Pushed a change to this and a corresponding change in annular_wedge.ts